widget/playground/src/containers/SupportedTokens/SupportedTokens.tsx

```diff
@@ -0,0 +1,69 @@
+import type { Type } from '../../types';
+
+import { ChainsIcon } from '@rango-dev/ui';
+import React from 'react';
+
+import { MultiSelect } from '../../components/MultiSelect/MultiSelect';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+import { tokensAreEqual } from '../../utils/common';
+
+export function SupportedTokens({ type }: { type: Type }) {
+  const {
+    config: { from, to },
+    onChangeTokens,
+    onChangeToken,
+  } = useConfigStore();
+  const {
+    meta: { tokens, blockchains },
+  } = useMetaStore();
+
+  const selectedType = type === 'Source' ? from : to;
+
+  const configTokens = selectedType?.tokens;
+
+  const seletedBlockchains = selectedType?.blockchains;
+  const allTokens = seletedBlockchains
+    ? tokens.filter((token) => seletedBlockchains.includes(token.blockchain))
+    : tokens;
+  const defaultBlockchains =
+    seletedBlockchains || blockchains.map((chain) => chain.name);
+
+  const list = allTokens.map((token) => {
+    return {
+      ...token,
+      checked:
+        !configTokens || configTokens.some((ct) => tokensAreEqual(ct, token)),
+    };
+  });
+
+  const isAllTokens = !configTokens || configTokens.length === list.length;
+
+  return (
```

Comment:
In some zooms, we have horizontal/vertical scrolling issue like this:

![Screenshot from 2023-11-05 13-08-36](https://github.com/rango-exchange/rango-client/assets/99180934/98a84654-6d96-420c-8890-960e1a012cd1)

Reply:
fixed.